postgres/pgdump.go

```diff
@@ -306,6 +314,15 @@ func processInsertStmt(conv *internal.Conv, n nodes.InsertStmt) *copyOrInsert {
 		logStmtError(conv, n, fmt.Errorf("can't get table name: %w", err))
 		return nil
 	}
+	if _, ok := conv.SrcSchema[table]; !ok {
```

Comment:
When I first saw this, I missed that you had added code into processCreateStmt to drop tables with INHERITS.
But now I see that code, this makes sense.

Suggest small tweak in the comment:

		// If we don't have schema information for a table, we drop all insert
		// statements for it. The most likely reason we don't have schema information
		// for a table is that it is an inherited table - we skip all inherited tables.

And something similar for COPY and ALTER.

Reply:
updated comment